### PR TITLE
PC-343: Add to confirm and submit checkbox text

### PR DESCRIPTION
**Pull request by chidin194**
image

### CODE REVIEW

**Review by CheshireSwift — CHANGES_REQUESTED** [review]
Couple of fixes required to the HTML. Haven't checked copy in detail.

File: help_to_heat/templates/frontdoor/confirm-and-submit.html [hunk 1]
```diff
@@ -25,10 +25,16 @@ <h1 class="govuk-heading-l">Confirm and submit</h1>
               support from the Great British Insulation Scheme and to contact me about my referral.
               I agree that they may share my personal details with their approved delivery partners and installers to check
               my eligibility, my property's suitability, and to contact me.
+
+                <p><a class="govuk-link" href="#">Further information on our GDPR UK Privacy Policy.</a></p>
```
[the remark this answers is not in the document]
Comment (CheshireSwift) [reply help_to_heat/templates/frontdoor/confirm-and-submit.html:29]: Yeah, I suspect it's meant to be a link somewhere at some point, but where might not be decided yet.

File: help_to_heat/templates/frontdoor/confirm-and-submit.html [hunk 2]
```diff
@@ -25,10 +25,16 @@ <h1 class="govuk-heading-l">Confirm and submit</h1>
...
+                <span class="govuk-heading-s">What is the Energy Company Obligation?</span>
```
[the remark this answers is not in the document]
Comment (CheshireSwift) [reply help_to_heat/templates/frontdoor/confirm-and-submit.html:30]: Per the design system:

> But change it if your pages feel unbalanced – the heading class you use does not always need to correspond to the heading level.

So I think it's okay to be whichever size we feel like. Given goals of:

* Matching the design.
* Having correct heading nesting.
* Matching the style to the tag.

I'm happy to break the third one in the interest of preserving the other two 🙂

File: help_to_heat/templates/frontdoor/confirm-and-submit.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (chidin194) [reply help_to_heat/templates/frontdoor/confirm-and-submit.html]: Confirmed with Allison it should redirect to the Privacy Policy page, added href in

File: help_to_heat/templates/frontdoor/confirm-and-submit.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (chidin194) [reply help_to_heat/templates/frontdoor/confirm-and-submit.html]: Matched design, thanks for explaining :)

**Review by CheshireSwift — APPROVED** [review]

**Pull request merged**